modules/checkout/commerce_checkout.plugin_type.yml

```diff
@@ -0,0 +1,12 @@
+commerce_checkout.checkout_pane:
+  label: Commerce checkout pane
+  provider: commerce_checkout
+  plugin_manager_service_id: plugin.manager.commerce_checkout_pane
+  plugin_configuration_schema_id: commerce_checkout.commerce_checkout_pane.[id]
+  plugin_definition_decorator_class: Drupal\commerce_checkout\PluginDefinition\CheckoutPanePluginDefinitionDecorator
```

Comment:
As there is nothing special about our plugins we can just provide the default one from plugin. Updated